packages/runtime/src/runClientApp.tsx

```diff
@@ -112,8 +112,17 @@ async function render({ history, runtime }: RenderOptions) {
   const RouteWrappers = runtime.getWrappers();
   const AppRouter = runtime.getAppRouter();
 
+  const rootId = appConfig.app.rootId || 'app';
+  let root = document.getElementById(rootId);
+  if (!root) {
+    root = document.createElement('div');
+    root.id = rootId;
+    document.body.appendChild(root);
+    console.warn('The root node is created by framework.');
```

Comment:
Root node #${rootId} is not found, current root is automatically created by the framework.